tasmota/language/fr_FR.h

```diff
@@ -57,9 +57,9 @@
 #define D_AS "comme"
 #define D_AUTO "AUTO"
 #define D_BATT "Batt"                // Short for Battery
-#define D_BLINK "Blink"
+#define D_BLINK "Blink"              // Not better in french
```

Comment:
I agree

Reply:
... for flash. Clignotement is not efficient, too verbose.

Reply:
Finally, I think that Flash may be misleading with Flash ROM, it's not accurate neither because a flash is a pulse of light, not an alternating state. It's mentioned in the Timer driver.